qualtran/bloqs/hubbard_model.py

```diff
@@ -303,12 +303,14 @@ def signature(self) -> Signature:
     def decompose_from_registers(
         self, *, context: cirq.DecompositionContext, **quregs: NDArray[cirq.Qid]
     ) -> Iterator[cirq.OP_TREE]:
+        from qualtran.resource_counting.symbolic_counting_utils import acos, ssqrt
```

Comment:
Will have to update your imports if #941  goes in first.

Reply:
fixed